src/main/java/cuchaz/enigma/gui/GuiController.java

```diff
@@ -230,6 +230,12 @@ public void rename(EntryReference<Entry<?>, Entry<?>> reference, String newName,
 		refreshCurrentClass(reference);
 	}
 
+	public void changeDocs(EntryReference<Entry<?>, Entry<?>> reference, String updatedDocs) {
+		this.deobfuscator.changeDoc(reference.getNameableEntry(), updatedDocs);
+
+		refreshCurrentClass(reference, true);
```

Comment:
Hm, is not ideal to have to re-decompile the class each time the doc changes. But I suppose without major changes that wouldn't really be avoidable?